vanilla/model_views.py

```diff
@@ -10,6 +12,9 @@
 import warnings
 
 
+text = unicode if sys.version_info[0] == 2 else str
```

Comment:
Use `from django.utils import six` and `six.text_type` in the code, rather than rebuild `six` one lien at time?